packages/mongo-nav/src/mongo-select/Trigger.tsx

```diff
@@ -67,25 +72,44 @@ export function OrganizationTrigger({
   className,
   urls,
   isActive,
+  disabled = false,
   ...rest
 }: OrganizationTriggerProps) {
   return (
     <div className={cx(orgTriggerContainer, className)}>
-      <button {...rest} className={buttonContainer}>
-        <Icon size="small" glyph="Building" />
-        <span className={selectedStyle}>{placeholder}</span>
-        <Icon size="small" glyph="CaretDown" />
-      </button>
-      <a
-        href={urls.mongoSelect.orgSettings}
-        className={cx(anchorStyle, border)}
-        aria-label="settings"
+      <button
+        {...rest}
+        className={buttonContainer}
+        data-testid="org-trigger"
+        disabled={disabled}
       >
         <Icon
-          glyph={'Settings'}
-          fill={isActive ? uiColors.green.base : uiColors.gray.base}
+          className={cx({ [disabledStyle]: disabled })}
+          size="small"
+          glyph="Building"
         />
-      </a>
+        <span className={cx(selectedStyle, { [disabledStyle]: disabled })}>
+          {disabled ? 'All Organizations' : placeholder}
+        </span>
+        <Icon
+          className={cx({ [disabledStyle]: disabled })}
+          size="small"
+          glyph="CaretDown"
+        />
+      </button>
+      {!disabled && (
+        <a
+          href={urls.mongoSelect.orgSettings}
+          className={cx(anchorStyle, border)}
+          aria-label="settings"
+          data-testid="org-trigger-settings"
+        >
+          <Icon
+            glyph={'Settings'}
```

Comment:
We can just have this as a string rather than evaluating it as a JS statement:
```js
glyph='Settings'
```

Reply:
Oops, this is something I'm used to linters handling. No prob